projects/plugins/jetpack/extensions/shared/memberships/settings.js

```diff
@@ -138,6 +188,19 @@ export function NewsletterAccessRadioButtons( {
 							{ accessLabel }
 							{ reach }
 						</label>
+						{
+							// This adds a tier selector when:
+							// - the paid_subscribers option is selected
+							// - stripe is connected
+							// - there are newsletter plans (the component will
+							//   check for more than 1 plan)
+							// - this isn't a paywall block
+						 }
+						{ key === accessOptions.paid_subscribers.key &&
+							key === accessLevel &&
+							isStripeConnected &&
+							hasNewsletterPlans &&
+							! postHasPaywallBlock && <TierSelector onChange={ onChange }></TierSelector> }
```

Comment:
```suggestion
							&& <TierSelector onChange={ onChange }></TierSelector> }
```

If we want to use the same selector in the paywall block too, let's remove this line.